### DISCUSSION TITLE
Show users joined/parted on a per-channel basis

### DISCUSSION THREAD

**Discussion by jimbo1111** [Chatterino/chatterino2#3728] [Ideas]
The toggle for "show users joined/parted" works globally - you can have it on for all channels, or off for all channels (though only under 1k users).

This would be more useful if it were a toggle so you could have it on or off for specific channels.  If I am in a relatively large chat I don't need to know this and it just gets spammy, but for my own small chat I would like to have it turned on.

A toggle which allows it being on/off for individual channels or splits would allow people more flexibility with this feature.

**Comment by Felanbird** [comment]
The problem with this is this is not a Chatterino feature, it's an IRC standard that Twitch supplies us, the 1k users being included in this fact.

The workaround approach to this would be using filters to remove the messages, which can be done with the following filter:
`!(flags.system_message && (message.content contains "Users joined" || message.content contains "Users parted"))`

![image](https://user-images.githubusercontent.com/41973452/168410452-3b69faa3-43b5-4f7a-ad4d-b9f87d2670f2.png)

**Reply by jimbo1111** [reply]
So could this not be added, then, as a standard feature, a checkbox in the settings? Since the framework for it is already there?

Or rather, allow this filter to be on by default and only turned off on a per-channel basis.  Currently it seems like you would have to add a custom filter for every single channel except for the one that you want to see join/part for - if the opposite could be done, and join/part messages could be ignored in channels by default and only shown in a few channels, that would be useful.
